platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_platform.sh

```diff
@@ -239,6 +239,10 @@ if [[ "$1" == "init" ]]; then
         /usr/local/bin/platform_watchdog_disable.sh
     fi
 
+    #Enable watcdog with nowayout
+    rmmod iTCO_wdt
+    modprobe iTCO_wdt nowayout=1
+
```

Comment:
- Added nowayout support.
- So, once watchdog is started we can't stop it. 
```
When the device is closed, the watchdog is disabled, unless the "Magic
Close" feature is supported (see below).  This is not always such a
good idea, since if there is a bug in the watchdog daemon and it
crashes the system will not reboot.  Because of this, some of the
drivers support the configuration option "Disable watchdog shutdown on
close", CONFIG_WATCHDOG_NOWAYOUT.  If it is set to Y when compiling
the kernel, there is no way of disabling the watchdog once it has been
started.  So, if the watchdog daemon crashes, the system will reboot
after the timeout has passed. Watchdog devices also usually support
the nowayout module parameter so that this option can be controlled at
runtime.

https://www.kernel.org/doc/Documentation/watchdog/watchdog-api.txt
```

Reply:
The platform API provides a function to "disarm" the watchdog. I'm not sure how frequently (or even if) this will be called. However, with "nowayout" enabled, it appears that there is no way to disable the watchdog after it has started. Is this correct?

Reply:
- Yes. We can't able to stop the watchdog once we enable with "nowayout" option. 
- The need for nowayout is if user-space watchdog daemon got crashed and accidentally if it close the /dev/watchdog node proerly then there is a possibility watchdog may never kick-in.
- To avoid this even slightest possibility this option is used.